platforms/native/testing-native/src/main/java/org/gradle/nativeplatform/test/cpp/plugins/CppUnitTestPlugin.java

```diff
@@ -138,12 +141,13 @@ private Stream<DefaultCppTestExecutable> getAllTestExecutable() {
                 task.setDescription("Executes C++ unit tests.");
 
                 final InstallExecutable installTask = binary.getInstallTask().get();
-                task.onlyIf("Test executable installation directory exists", element -> binary.getInstallDirectory().get().getAsFile().exists());
+                DirectoryProperty installDirectory = binary.getInstallDirectory();
```

Comment:
Fixes for CC compatibility.

Reply:
💡 After extracting the `installDirectory` property, the lambda could have been kept via `SerializableLambdas.spec(...)`